docs/stencil-docs/installing-stencil-cli/installing-stencil.mdx

```diff
@@ -6,9 +6,9 @@ BigCommerce is currently targeting 11/1/2023 to sunset its node-sass fork in fav
 </Callout>
 
 
-Stencil CLI gives developers the power to locally edit and preview themes with no impact to a merchant’s live storefront, and it's built-in [Browsersync](https://github.com/bigcommerce/browser-sync) capabilities make simultaneous testing across desktop, mobile, and tablet devices a breeze. Once work is complete, developers can push themes to BigCommerce storefronts (and set them live) using Stencil CLI's simple, yet powerful commands.
+Stencil CLI gives developers the power to locally edit and preview themes with no impact to a merchant's live storefront, and its built-in [Browsersync](https://github.com/bigcommerce/browser-sync) capabilities make simultaneous testing across desktop, mobile, and tablet devices a breeze. Once work is complete, developers can push themes to BigCommerce storefronts and make them live using Stencil CLI's simple yet powerful commands.
 
-This article contains the detailed instructions needed to install and configure Stencil CLI -- the first step to developing themes on the BigCommerce platform.
+This article contains the detailed instructions needed to install and configure Stencil CLI - the first step to developing themes on the BigCommerce platform.

```

Comment:
```suggestion
This article contains detailed instructions to install and configure Stencil CLI - the first step to developing themes on the BigCommerce platform.

```

Reply:
i like this and want to take it further.

```suggestion
This article contains detailed instructions on installing and configuring Stencil CLI, the first step towards developing Stencil themes for BigCommerce storefronts.

```